src/python/ensembl/core/models.py

```diff
@@ -2020,3 +2021,9 @@ def compile_longtext_sqlite(type_, compiler, **kw):  # pylint: disable=unused-ar
 def compile_tinyint_sqlite(type_, compiler, **kw):  # pylint: disable=unused-argument
     """Cast MySQL TINYINT to SQLite INT."""
     return "INT"
+
+
+@compiles(DOUBLE, "sqlite")
+def compile_double_sqlite(type_, compiler, **kw):  # pylint: disable=unused-argument
```

Comment:
```suggestion
def compile_double_sqlite(
    type_: sqlalchemy.sql.expression.ColumnClause,  # pylint: disable=unused-argument
    compiler: sqlalchemy.engine.interfaces.Compiled,  # pylint: disable=unused-argument
    **kw: Any,  # pylint: disable=unused-argument
) -> str:
```
Just to align with the other type hints.